### ISSUE TITLE
Package Manager for Plugins/External deps

### ISSUE THREAD

**Issue by emoon** [emoon/ProDBG#148]
Something similar to Sublimes package manager which is quite nice and integrated. Hopefully usable for other things as well (like prebuilt libs)

**Comment by kondrak** [comment]
Sounds like something that could be potentially done with Cargo, now that ProDBG is moving to Rust?

**Comment by emoon** [comment]
No, I'm thinking more of binary and/or scripts. Say that someone wants to publish a plugin for ProDBG it can be written in either C/Rust/C++ and in the future Lua/JS or whatever lang ProDBG will support.
